src/frontend/components/SelectContent/index.tsx

```diff
@@ -421,6 +433,23 @@ export const SelectContent = ({
       </form>
 
       <Tabs>
+        <Tab title="Webinars">
+          <SelectContentSection
+            addMessage={messages.addWebinar}
+            addAndSelectContent={async () => {
+              useCreateVideoMutation.mutate({
+                playlist: playlist!.id,
+                title: lti_select_form_data?.activity_title,
+                description: lti_select_form_data?.activity_description,
+                live_type: LiveModeType.JITSI,
```

Comment:
As you are calling `initiateLive` once created, is it needed to specify the `live_type` here ?

Reply:
I did it this way to match how it is done through the existing app.
Also, live_type parameter is used in the success callback to call `initateLive`
https://github.com/openfun/marsha/pull/1592/files/50c5d968f1114aab3c5d2a25667b09573250c86e#diff-a57ea6d3b5fea5302c3087db3f99786eb674ca04bb5b6b72d3d8419dbdded9b9R314